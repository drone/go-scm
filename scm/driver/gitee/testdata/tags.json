[
  {
    "name": "release-test",
    "message": "",
    "commit": {
      "sha": "25b13a79d771314d0075302178eb18aef79e80db",
      "date": "2021-08-17T16:01:03+00:00"
    }
  },
  {
    "name": "1.1",
    "message": "1.1",
    "commit": {
      "sha": "5e7876efb3468ff679410b82a72f7c002382d41e",
      "date": "2021-03-24T11:29:35+08:00"
    }
  },
  {
    "name": "1.0",
    "message": "1.0",
    "commit": {
      "sha": "5e7876efb3468ff679410b82a72f7c002382d41e",
      "date": "2021-03-24T11:29:35+08:00"
    }
  },
  {
    "name": "1.2",
    "message": "",
    "commit": {
      "sha": "5e7876efb3468ff679410b82a72f7c002382d41e",
      "date": "2021-03-24T11:29:35+08:00"
    }
  }
]