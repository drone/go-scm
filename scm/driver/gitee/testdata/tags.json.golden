[
  {
    "Name": "release-test",
    "Path": "refs/tags/release-test",
    "Sha": "25b13a79d771314d0075302178eb18aef79e80db"
  },
  {
    "name": "1.1",
    "Path": "refs/tags/1.1",
    "Sha": "5e7876efb3468ff679410b82a72f7c002382d41e"
  },
  {
    "name": "1.0",
    "Path": "refs/tags/1.0",
    "Sha": "5e7876efb3468ff679410b82a72f7c002382d41e"
  },
  {
    "name": "1.2",
    "Path": "refs/tags/1.2",
    "Sha": "5e7876efb3468ff679410b82a72f7c002382d41e"
  }
]